core/src/net/sf/openrocket/rocketcomponent/Parachute.java

```diff
@@ -9,24 +9,31 @@
 
 public class Parachute extends RecoveryDevice {
 	private static final Translator trans = Application.getTranslator();
-	
-	public static double DEFAULT_CD = 0.8;
-	
+	private final double defaultDiameter = 0.3;
 	private double diameter;
+	public static double DEFAULT_CD = 0.8;
+	private final Material defaultLineMaterial;
+	private Material lineMaterial;
+	private final int defaultLineCount = 6;
+	private int lineCount = defaultLineCount;
+	private final double defaultLineLength = 0.3;
```

Comment:
You know the drill --> DEFAULT_LINE_LENGTH

Reply:
Changed.